triggers/runner/src/main/scala/com/digitalasset/daml/lf/engine/trigger/Converter.scala

```diff
@@ -287,7 +296,7 @@ final class Converter(
       SUnit,
     )
 
-  def fromACS(createdEvents: Seq[CreatedEvent]): Either[String, SValue] =
+  def fromActiveContracts(createdEvents: Seq[CreatedEvent]): Either[String, SValue] =
```

Comment:
Renamed as the created `Value` is actually an instance of the Daml type `ActiveContracts`